Tests/BSONTests/BSONCorpusTests.swift

```diff
@@ -215,10 +212,6 @@ final class BSONCorpusTests: BSONTestCase {
 
             if let decodeErrors = testFile.decodeErrors {
                 for test in decodeErrors where shouldRun(testFile.description, test.description) {
-                    let description = "\(testFile.description)-\(test.description)"
```

Comment:
I think this description still needs to be defined for the `XCTFail` below to compile

Reply:
Oops! good catch turning the git hook back on 😅